### ISSUE TITLE
Implement ConvertID function

### ISSUE THREAD

**Issue by VanadiumJade** [mortenbra/alexandria-plsql-utils#64]
It would be awesome if the ConvertID function was included in package.

I would like to convert EWS ID into OWA ID - so I can create a OWA link so users can be directed directly to web email.

**Comment by mortenbra** [comment]
Did some research to find the relevant documentation:

https://docs.microsoft.com/en-us/exchange/client-developer/web-service-reference/convertid-operation

https://docs.microsoft.com/en-us/exchange/client-developer/exchange-web-services/ews-identifiers-in-exchange

**Comment by VanadiumJade** [comment]
Yeah, I have spent significant time trying to get something working.

I have tried to add a function into your package, but the lack of detail in the convertID API makes it quite hard. Getting the item ews id in a format that the API will accept. Best I have been able to do is get a malformed ewsid response or error in ewsid.

Any help here would be much appreciated!
